### PR TITLE
SCRUM-52-DELETE menssages

### PR DESCRIPTION
**Pull request by ErickaIsa**
Add code for my test cases
Added test cases:
test_delete_email_important_success`: Verifies successful deletion of a single important email.
test_delete_multiple_important_emails`: Tests deletion of multiple important emails.
test_delete_email_important_without_auth_header`: Checks unauthorized deletion without auth header.
test_delete_email_important_invalid_auth_token`: Ensures unauthorized deletion with invalid token.
test_delete_email_important_check_response_content_type`: Validates response content type after deletion.
For the moment, I found 5 bugs
image

### CODE REVIEW

**Review by Miguel-Lopez004 — APPROVED** [review]

**Review by Jasonn5 — APPROVED** [review]
Reviewed, executed, work as image description expectations of PR, good job Ericka

File: config/config.py
Comment (AlisonGuzman2006) [inline config/config.py:1]: The code addition has been reviewed and accepted. Good Job Ericka!

File: tests/mail_important/conftest.py
Comment (AlisonGuzman2006) [inline tests/mail_important/conftest.py:1]: The changes are reviewed and accepted!! Everything is working correctly.

File: tests/mail_important/test_delete_email_important.py
Comment (AlisonGuzman2006) [inline tests/mail_important/test_delete_email_important.py:1]: The new code has been reviewed and accepted!! Everything is working correctly.

**Review by AlisonGuzman2006 — APPROVED** [review]
The changes nad new code addition have been reviewed and accepted!! Everything is working correctly. Great work!!

**Pull request merged**